zebra/label_manager.c

```diff
@@ -145,6 +147,20 @@ void lm_hooks_unregister(void)
 	hook_unregister(lm_release_chunk, label_manager_release_label_chunk);
 }
 
+DEFPY(show_label_table, show_label_table_cmd, "show debugging label-table",
```

Comment:
DEFPY should be formatted as other commands, token per new line:

```
SHOW_STR
DEBUG_STR
"Display allocated label chunks\n"
```

Reply:
Done.